solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -938,7 +922,7 @@ Acknowledgements may throw `SolaceAcknowledgmentException` depending on the curr
 *Example:* +
 (refer to <<Message Redelivery>> for background info)
 
-A `SolaceAcknowledgmentException` with cause `SolaceStaleMessageException` may be thrown when trying to asynchronously `ACCEPT` a stale message after the timeout elapses for the `REQUEUE` of another message. Though for this particular example, since the message that failed to `ACCEPT` will be redelivered, this exception can be caught and ignored if you have no business logic to revert.
+A `SolaceAcknowledgmentException` with cause `SolaceStaleMessageException` may be thrown when trying to asynchronously `ACCEPT` a stale message. Though for this particular example, since the message that failed to `ACCEPT` will be redelivered, this exception can be caught and ignored if you have no business logic to revert.
```

Comment:
Does the `SolaceStaleMessageException` apply to `REJECT` and `REQUEUE` as well now that we're doing native message settlements?

Reply:
Done.